File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Pyramid Height: ")  # ask user for height
    if height > 0 and height <= 8:  # height has to be between 1 and 8 inclusive
        break

for i in range(height):
    spaces = height - i - 1  # print spaces
    bricks = i + 1  # print bricks
    for k in range(spaces):
        print(" ", end="")
    for j in range(bricks):
        print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask user for positive whole number
    int height;
    do
    {
        height = get_int("Please give a whole number greater than 0: ");
    }
    while (height < 1);

    // print brick pyramid
    for (int i = 0; i < height; i++)
    {
        // print row
        for (int k = height - i; k > 1; k--) // spaces
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++) // bricks
        {
            printf("#");
        }
        printf("\n");
    }
}
